include/foonathan/memory/detail/ebo_storage.hpp

```diff
@@ -5,8 +5,8 @@
 #ifndef FOONATHAN_MEMORY_DETAIL_EBO_STORAGE_HPP_INCLUDED
 #define FOONATHAN_MEMORY_DETAIL_EBO_STORAGE_HPP_INCLUDED
 
-#include "detail/utility.hpp"
-#include "config.hpp"
+#include "utility.hpp"
+#include "../config.hpp"
```

Comment:
Yes, noticed that one as well, is fixed in a branch currently on CI.

Reply:
ok, will be a small merge conflict probably